testsuite/testsuite-jose-jose4j/src/test/resources/project-jwk-inline-store.yml

```diff
@@ -0,0 +1,31 @@
+swarm:
+  jose:
+    keystore:
+      path: inline
+      type: jwk
+      jwkset: |
+        {
+          "keys":
```

Comment:
to be sure I understand it, there's one key for encryption and one for decryption and on a specific operation the proper key is selected based on `alias == kid`?